src/LoginPage.cpp

```diff
@@ -153,7 +151,6 @@ LoginPage::LoginPage(QWidget *parent)
 
         error_label_ = new QLabel(this);
         error_label_->setFont(font);
-        error_label_->setWordWrap(true);
```

Comment:
This is needed, since otherwise some errors are too long and get cut off. In those cases you can't read the important parts of the error message.

Reply:
But there are problem with hint, because It's print just "You have entered an invalid Matrix ID", and it's not clear how Matrix ID should looks. Maybe better to add a new label for this matter?